src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/ContentUpdatesForm.js

```diff
@@ -71,23 +72,41 @@ export const ContentUpdatesForm = ({
   const handleChange = ({ rowIndex, actionIndex, value, fieldName }) => {
     setFields(fieldsArr => fieldsArr.map((field, i) => {
       if (i === rowIndex) {
+        const hasActionChanged = fieldName === ACTION_VALUE_KEY;
+
+        let actions = field.actionsDetails.actions.map((action, j) => {
+          if (!action) return action; // if null, return this value to stay with the same arr length
+
+          return j === actionIndex
```

Comment:
In my opinion it will looks better in separate semanticly named function

Reply:
Makes sense, moved into functions